doc/10-icinga-template-library.md

```diff
@@ -5939,6 +5939,7 @@ ssl_cert_ignore_ocsp_errors   | **Optional.** Continue if the OCSP status cannot
 ssl_cert_ignore_ocsp_timeout  | **Optional.** Ignore OCSP result when timeout occurs while checking.
 ssl_cert_ignore_sct           | **Optional.** Do not check for signed certificate timestamps.
 ssl_cert_ignore_tls_renegotiation  | **Optional.** Do not check for renegotiation.
+ssl_cert_dane                 | **Optional.** Verify that valid DANE records exist {211,301,302,311 or empty string}.
```

Comment:
```suggestion
ssl_cert_dane                 | **Optional.** Verify that valid DANE records exist (211, 301, 302, 311 or empty string).
```

Reply:
Aesthetically, I'm with you. 

Same file, a couple of lines further up:
```
ssl_cert_ssl_version          | **Optional.** Force specific SSL version out of {ssl2,ssl3,tls1,tls1_1,tls1_2}.
ssl_cert_disable_ssl_versions | **Optional.** Disable specific SSL versions out of {ssl2,ssl3,tls1,tls1_1,tls1_2}. Multiple versions can be given as array.
```

I tried to follow suit.

Reply:
Then it's

```suggestion
ssl_cert_dane                 | **Optional.** Verify that valid DANE records exist ({211,301,302,311} or empty string).
```

IMAO